Sources/DataPipeline/DataPipelineConfigOptions.swift

```diff
@@ -26,6 +26,7 @@ public struct DataPipelineConfigOptions {
         public static func create(sdkConfig: SdkConfig) -> DataPipelineConfigOptions {
             let writeKey = "\(sdkConfig.siteId):\(sdkConfig.apiKey)"
             var result = DataPipelineConfigOptions(writeKey: writeKey)
+            result.autoTrackDeviceAttributes = sdkConfig.autoTrackDeviceAttributes
```

Comment:
The test device attributes was breaking we now use `DataPipelineConfigOptions` and it was not copied from `SdkConfig` mistakenly

Reply:
Do we have a test for this `DataPipelineConfigOptions.create(sdkConfig)` function? Looks like one could be valuable to catch bugs like this where we forget to add a `result.N = sdkConfig.N` line.

Reply:
Unfortunately, we don't have a test for this function yet. I would love to add one though, but I'm unsure about the type of test that would ensure its reliability and fail every time we add a property to `DataPipelineConfigOptions` unless we specify that it's not from `SdkConfig`. If you have any ideas, please do share, and I can then work on adding the necessary tests.